coredns/tests/common.py

```diff
@@ -81,6 +81,8 @@
 
 METRICS_V1_7 = COMMON_METRICS + [
     NAMESPACE + '.forward_max_concurrent_rejects',
+    NAMESPACE + '.forward_conn_cache_hits',
+    NAMESPACE + '.forward_conn_cache_misses',
```

Comment:
The tests are failing on these metrics. It looks like they were added in [1.7.1](https://github.com/coredns/coredns/blob/v1.7.1/plugin/forward/README.md), however our e2e tests version 1.7.0. Can you try removing them from this list? The tests should pass. Since you added them to the `fixtures/metrics.txt`, they'll still get tested.